foreman_maintain.gemspec

```diff
@@ -24,6 +24,10 @@ the Foreman/Satellite up and running."
   s.add_dependency 'clamp'
   s.add_dependency 'highline'
 
+  if RUBY_VERSION >= '3.0'
+    s.add_dependency 'rexml' # rexml was a default gem in ruby < 3.0, and is a bundled gem in >= 3.0
+  end
```

Comment:
Sadly you can't do this in a gemspec. AFAIK the gemspec is evaluated during `gem build` but not after. So that's what's then uploaded to Rubygems. I'm not sure how you're supposed to deal with it.

Reply:
ewww.

well, we could add the same logic to `Gemfile`?

Reply:
opn the other hand, adding the rexml dependency on pre 3.0 doesn't hurt anyone

Reply:
I don't know if it doesn't hurt anyone. That said, Ruby 3 support is still a long way out for Foreman itself. It needs some serious work on fog which is a deep rabbit hole.

Reply:
it's bundled in < 3.0, so depending on it shouldn't change anything.

I can remove ruby3 from this PR, but this also means I can't run FM tests on my Fedora :(

Reply:
I'd recommend adding it to Gemfile for now then.

Reply:
done